docs/setup/install/install-devtron.md

```diff
@@ -1,24 +1,20 @@
-# Install Hyperion using Helm3
```

Comment:
Why these changes? We can just mention `Deprecated` and tell the users that Hyperion is now Devtron.